### PR TITLE
You can now make universal enzyme in chemistry.

### LINKED ISSUE #8360: Universal enzyme has no recipe
There is no recipe for obtaining the universal enzyme.

### PR DESCRIPTION
**Pull request by Intigracy**
fixes #8360

tested-stamp

### CODE REVIEW

**Comment by ihadtoregisterforthis** [comment]
App Deyt Wee Ki

**Comment by Intigracy** [comment]
I will after it's merged.

**Comment by 9600bauds** [comment]
Personally, I'm unsure but w/e

**Comment by Intigracy** [comment]
The only other way to acquire it besides the roundstart bottle in the kitchen is through cargo.

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 1]
```diff
@@ -1852,6 +1852,13 @@ datum
 
 //////////////////////////////////////////FOOD MIXTURES////////////////////////////////////
 
+		enzyme
+			name = "Universal Enzyme"
+			id = "enzyme"
+			result = "enzyme"
+			required_reagents = list("sodiumchloride" = 1, "potassium" = 1, "sugar" = 1)
```
Comment (9600bauds) [inline code/modules/reagents/Chemistry-Recipes.dm]: At least have it need Discount Dan's or something, muh immulsions

Comment (clusterfack) [inline code/modules/reagents/Chemistry-Recipes.dm]: Have it need soylent green

Comment (Intigracy) [inline code/modules/reagents/Chemistry-Recipes.dm]: is soylent green a chem too, or just a food?

**Comment by Shadowmech88** [comment]
The recipe doesn't make a whole lot of sense.

**Comment by Intigracy** [comment]
I'm open to suggestions.

Soylent green isn't a reagant, it's a food.

**Comment by Intigracy** [comment]
Edited recipe.

Moonshine change is so you don't accidentally make moonshine while making enzyme.

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 2]
```diff
@@ -2033,7 +2040,7 @@ datum
 			name = "Moonshine"
 			id = "moonshine"
 			result = "moonshine"
-			required_reagents = list("nutriment" = 10)
+			required_reagents = list("cornoil" = 10)
```
Comment (9600bauds) [inline code/modules/reagents/Chemistry-Recipes.dm:2043]: I hope you realize this is a massive inconvenience

Comment (Intigracy) [inline code/modules/reagents/Chemistry-Recipes.dm:2043]: It's grinding corn.

Nobody makes moonshine anyway as-is, because enzyme is so hard to come by.

Comment (Intigracy) [inline code/modules/reagents/Chemistry-Recipes.dm:2043]: In addition, most every other alchohol in this file requires something from botany.

wine needs berryjuice
spacebeer needs flour
vodka needs potato
sake needs rice

**Comment by Intigracy** [comment]
Unless someone has a better idea for the recipe, this is good to go.

I'll update the wiki on merge.

**Comment by Shadowmech88** [comment]
Enzymes are biological in nature, so a mixture of different fruit juices might be more appropriate. Additionally, cardiac enzymes can be present in blood at low levels.
There are many enzymes present in milk as well.

**Comment by Intigracy** [comment]
wiki updated

**Comment by 9600bauds** [comment]
firefox_2016-04-12_21-44-28
n i c e

**Comment by Intigracy** [comment]
eww

**Comment by Intigracy** [comment]
fixed

**Comment by 9600bauds** [comment]
:+1:

**Pull request merged**